rwengine/src/objects/CharacterObject.cpp

```diff
@@ -25,6 +25,8 @@
 #include "engine/GameWorld.hpp"
 #include "loaders/LoaderIFP.hpp"
 #include "objects/VehicleObject.hpp"
+#include "CharacterObject.hpp"
```

Comment:
I'm not sure if this header is needed.

Reply:
Ah damn I missed that. CLion does this whenever I use the `add implementation` shortcut.

Reply:
qt creator <3